Makefile

```diff
@@ -86,14 +86,18 @@ e2e-run-all: ## Run E2E tests: all
 	$(E2E_RUN) npm run e2e:all
 
 
+psql-shell: ## Assuming a system is already running with `make start`, start up an interactive psql shell
+	docker-compose exec postgres psql --username postgres --dbname polis-dev
```

Comment:
@ballPointPenguin Should this actually be `--db-name polis-$TAG`? Not sure how `dev` gets set, but assuming that's not right for production connections...

Reply:
Maybe it should pull DB_NAME from env

Reply:
it's POSTGRES_DB in env.